_posts/2021-08-29-mosaic-fusion.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Mosaic Image Fusion
+author: Volker Hilsenstein, Marvin Albert, and Genevieve Buckley
```

Comment:
Hi @m-albert - just making you aware that you're listed as a co-author of this blogpost, Volker tells me your contributions to the project were substantial.